src/components/TransactionsTab/subcomponents/ClearTransactionsSelector/ClearTransactionsSelector.styles.tsx

```diff
@@ -0,0 +1,40 @@
+import styled from "styled-components/macro";
+
+import { Tooltip } from "../../../../styled-components/Tooltip/Tooltip";
+import Dropdown from "../../../Dropdown/Dropdown";
+import IconButton from "../../../IconButton/IconButton";
+
+export const Container = styled.div`
+  position: relative;
+`;
+
+export const StyledDropdown = styled(Dropdown)`
+  position: absolute;
+  top: 2.125rem;
+  right: 2.125rem;
+  min-width: 5rem;
+`;
+
+export const ClearListButtonTooltip = styled(Tooltip)`
```

Comment:
`z-index: 1` fixes the overlapping issue

![Screenshot 2023-07-25 at 4 24 36 PM](https://github.com/airswap/airswap-web/assets/90465420/edeadc41-7519-4a06-9d6a-90169c4616ba)